shiny/_static.py

```diff
@@ -24,208 +22,38 @@ class FileContentJson(TypedDict):
 
 
 def deploy_static(
-    appdir: Union[str, Path],
+    appdirs: Tuple[Union[str, Path]],
```

Comment:
Is this type annotation right? Doesn’t this restrict you to a tuple
of length 1?

Reply:
Good catch, thanks.